src/inc/til/winrt.h

```diff
@@ -8,12 +8,12 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
     template<typename T>
     struct property
     {
-        explicit constexpr property(auto&&... args) :
+        explicit constexpr property(auto&&... args) noexcept :
             _value{ std::forward<decltype(args)>(args)... } {}
 
         property& operator=(const property& other) = default;
 
-        T operator()() const noexcept
+        T operator()() const noexcept(noexcept(static_cast<T>(_value)))

```

Comment:
I believe this should be `std::is_nothrow_copy_constructible`, but I think it would be even more ideal for us to just return `const T&` here and leave the unconditional `noexcept`.

Reply:
returning `const T&` doesn't work with `winrt::hstring`, so went with the `std::is_nothrow_copy_constructible` solution, thanks!